### PR TITLE
docs: introduce serve docs

### PR DESCRIPTION
**Pull request by turbo1912**

### CODE REVIEW

File: docsite/docs/fal-serverless/isolated_functions/serving.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+You'll receive an application ID in the following format: \`XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX\`.
+
+## Access Served Function via REST API
+
+To access the served function, make a POST REST API request to the following URL:
+
+```
+https://<userid>-<application_id>.gateway.alpha.fal.ai
```
Comment (drochetti) [inline docsite/docs/fal-serverless/isolated_functions/serving.md:35]: Is `alpha` correct here?

File: docsite/docs/fal-serverless/isolated_functions/serving.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+Replace `<userid>` with your user ID (e.g., `123` if your github id is `github|123`) and `<application_id>` with the application ID you received earlier. Additionally, pass your FAL key ID and key secret as headers in the request.
```
Comment (drochetti) [inline docsite/docs/fal-serverless/isolated_functions/serving.md:38]: iirc our CLI prints the URL, so maybe we could avoid this instruction to reduce complexity?

File: docsite/docs/fal-serverless/isolated_functions/serving.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 5
+---
+
+# Serving functions though a managed web server
+
+Serving an `@isolated` function exposes the function through a webserver managed by `fal-serverless`.
+
+Simplest way to serve a function is to mark the function you want to serve using the `@isolated` annotation's `serve=True` option and `fal-serverless` cli command.
```
Comment (drochetti) [inline docsite/docs/fal-serverless/isolated_functions/serving.md:9]: Should we stick to _decorator_ instead of _annotation_ when referring to `@isolated`?

**Comment by drochetti** [comment]
Thanks for doing this! I'll follow up with my js client docs that links nicely to this

File: docsite/docs/fal-serverless/isolated_functions/serving.md (hunk 1 above)
Comment (chamini2) [inline docsite/docs/fal-serverless/isolated_functions/serving.md:35]: I think we can just rely on fal-serverless to print the appropriate URL to use. We can also include the URL in the future `fal-severless functions list` command or a `fal-serverless functions info <func_id>` command

Comment (turbo1912) [reply docsite/docs/fal-serverless/isolated_functions/serving.md:35]: yup

File: docsite/docs/fal-serverless/isolated_functions/serving.md (hunk 2 above)
Comment (turbo1912) [reply docsite/docs/fal-serverless/isolated_functions/serving.md:38]: yes yes fixed it in the new revision

**Review by burkaygur — APPROVED** [review]

**Pull request merged**